infrasim/chassis/emu_data.py

```diff
@@ -179,8 +184,9 @@ def ChangeChassisInfo(self, info):
         # Refer to EMC technical white paper in Dell Community, the Type is 0x17
         result.append(0x17)
         self.__fill_table(result, _ori_values)
-
-        self._data_area[FruCmd.CHASSIS_INFO_AREA] = {"start": 0, "end": 0, "data": result}
+        if len(result) > len(_data):
```

Comment:
what will be filled here if the result is too long?

Reply:
nothing changed if new data is too long.